tests/README.md

```diff
@@ -32,20 +43,29 @@ for more information.
 
 Run storage tests:
 
-    tox -e storage
-
-Increasing test verbosity:
+    tox -e storage -- -vv --no-cov storage
 
-    tox -e storage -- -vv
+Note that for some test modules (like `storage` or `virt`), _if_ there
+are any extra pytest arguments (after the `--`) then the search path at
+the end needs to be specified explicitly, because the configuration in
+`tox.ini` doesn't take care of it in that case and too many tests would be
+collected.  
```

Comment:
It should be clarified what "search path" is, best accompanied with an example.  Also, "too many tests would be collected" sounds confusing -- can we run "too many" tests?

And there is an extra space after the last sentence.

Reply:
I'm afraid I don't know how to clarify that well because I'm not experienced enough with tox/pytest.

"too many tests" means "more tests than the user actually wanted to specify". If I think I was asking for the storage tests only, but then other tests run as well, because it doesn't work like that, then that is "too many tests". How could we say it better? Maybe "unintended tests"?

The two spaces after "collected." are for the line break.

Reply:
> "too many tests" means "more tests than the user actually wanted to specify". If I think I was asking for the storage tests only, but then other tests run as well, because it doesn't work like that, then that is "too many tests". How could we say it better? Maybe "unintended tests"?

"unintended test" would be definitely better; or maybe something like "tests out of the specified collection would be run" could be used (+ the example how to specify the path to work around the problem).

> The two spaces after "collected." are for the line break.

Ah, I see.

Reply:
"too many tests" can be replaced by "tests from other tox env".

Reply:
> (+ the example how to specify the path to work around the problem)

The example was:
`tox -e storage -- -vv --no-cov storage`
Do You think that is sufficient?

Reply:
Yes, I understand now what's meant there, thanks.